tidy3d/components/simulation.py

```diff
@@ -49,6 +57,8 @@
 MAX_CELLS_TIMES_STEPS = 1e17
 MAX_MONITOR_DATA_SIZE_BYTES = 10e9
 
+SMALL_NUM = 1e-5
```

Comment:
Can we be more specific about what this small number is doing?
* Add a comment
* Call it something more specific, like `DIST_NEIGHBOR_2D_MEDIUM` (?)

Otherwise from just reading this file, it might be confusing what this is doing below.